manifests/2/2dust/v2rayN/6.53/2dust.v2rayN.locale.zh-CN.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.6.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: 2dust.v2rayN
+PackageVersion: "6.53"
+PackageLocale: zh-CN
+Publisher: 2dust
+PublisherUrl: https://github.com/2dust
+PublisherSupportUrl: https://github.com/2dust/v2rayN/issues
+Author: 2dust
+PackageName: v2rayN
+PackageUrl: https://github.com/2dust/v2rayN
+License: GPL-3.0
+LicenseUrl: https://github.com/2dust/v2rayN/blob/master/LICENSE
+ShortDescription: A GUI client for Windows, support Xray core and v2fly core and others

```

Comment:
Hi @stevenlele,

I didn't block the PR on this, but shouldn't the `ShortDescription` be in the localization file's language?